_includes/display-logo.html

```diff
@@ -0,0 +1,36 @@
+{% comment %}
+Logo and screenshots handler.
+{% endcomment %}
+
+{% assign itemLogo = include.itemLogo %}
+{% assign screenshots = include.screenshots %}
+{% assign classes = include.classes %}
+{% assign codiceIPA = include.codiceIPA | default: "false" %}
+
+{% if itemLogo contains 'github' and itemLogo contains '.svg' %}
+  {% assign logo=itemLogo | append: '?sanitize=true' %}
+{% else %}
+  {% assign logo=itemLogo %}
+{% endif %}
+
+{% if codiceIPA != "false" %}
+{% assign placeholderScreen = "/assets/images/placeholder-software-riuso.jpg" %}
+{% else %}
+{% assign placeholderScreen = "/assets/images/cover_software_opensource.png" %}
+{% endif %}
```

Comment:
If we leave it like this without solving https://github.com/italia/developers.italia.it/pull/561#discussion_r426303850 first, inside the software-details page it will always default to `opensource` even when it's not. This is not good IMHO @sebbalex

Reply:
No, it's not good at all. I've opened [this](https://github.com/italia/developers-italia-backend/issues/150) track down.

Reply:
Great! Let's wait for that to be fixed before merging this. As such I am removing this from the current milestone and moving it to the next one, thanks @sebbalex!